src/tabs/serverAdmins/ServerAdminsTabState.tsx

```diff
@@ -3,7 +3,26 @@ export interface IdNamePair {
   name: string;
 }
 
+export interface Permissions {
```

Comment:
Maybe this could be moved to a separate file since it's also needed in the dialog? Just to keep the various components as separate as possible (in this case, moving `Permissions` and `AdminPermissions` to a separate file would free you from the necessity of [including `ServerAdminsTabState`, which is meant to only be used with the "server admins" tab, in `ManagePermissionsDialog`](https://github.com/MovingBlocks/FacadeServer-frontend/pull/7/files#diff-e74a649ef2046a49035ea5901a4ba74aR5)